packages/common/src/api/tan-query/useFeaturedPlaylists.ts

```diff
@@ -0,0 +1,20 @@
+import { useCollections } from './useCollections'
+import { useExploreContent } from './useExploreContent'
+
+type Config = {
```

Comment:
import config
if you need to add to the pick list, that should be okay since they're all optional